smart_contracts/ethereum/solidity_firefly/artifacts/contracts/Firefly.sol/Firefly.dbg.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Was it intentional to check in the "artifacts" and "cache" folders? Seems like they should be gitignored.